std/random.d

```diff
@@ -20,7 +20,7 @@ Example:
 ----
 // Generate a uniformly-distributed integer in the range [0, 14]
 auto i = uniform(0, 15);
-// Generate a uniformly-distributed real in the range [0, 100$(RPAREN)
+// Generate a uniformly-distributed real in the range [0, 100)
```

Comment:
Unfortunately, you can't work around [issue 15122](https://issues.dlang.org/show_bug.cgi?id=15122) that way.